client/app/scripts/superdesk/ui/ui.js

```diff
@@ -252,6 +252,29 @@ define([
 		};
 	}
 
+	DropdownPositionDirective.$inject = ['$document'];
+	function DropdownPositionDirective($document) {
+		return {
+			link: function(scope, element) {
+
+				var tolerance = 250;
+
+				element.bind('click', function(event) {

```

Comment:
I think that only those handlers attached to the element **diferent than the one linked to directive**, should be unbinded. For example if I attach `click` event to `document`, here,  in this directive, I would need to unbind it.